app/themes/user-admin/e2e/home/home.e2e-spec.ts

```diff
@@ -0,0 +1,53 @@
+import { UserAdminPage } from './home.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('Home Page', function() {
+    let page: UserAdminPage;
+
+    beforeEach(() => {
+        page = new UserAdminPage();
+    });
+
+    it('should login successfully with right email and password', () => {
+        page.navigateTo();
+        let email = browser.findElement(protractor.By.name('email'));
+        let password = browser.findElement(protractor.By.name('password'));
+        let submit = browser.findElement(protractor.By.className('btn btn-success'));
+
+        email.sendKeys('foo@mail.com');
```

Comment:
we need to have a consistent test username & password. please move this function into a page object/fixture so we only have once place to update a test user account.

also, please create an SQL script to add the test user (as some systems will not have this by default). put the sql in this directory: `user-admin/scripts/sql/create-e2e-test-user.sql`